docs/explanation/bases.rst

```diff
@@ -0,0 +1,67 @@
+.. _bases_explanation:
+
+Bases
+=====
+
+One mandatory field for every rock project is the
+:ref:`base <rockcraft_yaml_base>`. 
+
+This field defines the baseline system upon which the rock's contents shall be
+layered on. Only Ubuntu and ``bare``
+:ref:`bases <rockcraft_yaml_base>` are supported.
+
+Ubuntu bases
+------------
+
+When setting an Ubuntu
+:ref:`base <rockcraft_yaml_base>` for a rock, Rockcraft will pull the
+corresponding Ubuntu base container image from a container registry. This is
+exactly the same container one would get from places like
+`Docker Hub <docker_hub_>`_ or `Amazon ECR <ecr_>`_.
+
+The resulting rock will therefore also contain and software and utilities that
+are present in the Ubuntu base container images.
+
+Setting an Ubuntu :ref:`base <rockcraft_yaml_base>` for a rock is
+especially useful when the goal is to build a rock that can be used for
+general-purpose environments such as a development working space.
+
+Bare bases
+----------
+
+As the name says, a ``bare`` :ref:`base <rockcraft_yaml_base>` indicates
+that the rock shall have no
+baseline system. This definition is similar to the "`scratch`_" Docker image,
+with the exception that a rock **can never be completely empty**, since it must
+always have :ref:`Pebble <pebble_explanation_page>` as its entrypoint, and
+include their own :ref:`metadata <what-sets-rocks-apart>` inside.
+
+When and how to use a ``bare`` base
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``bare`` :ref:`base <rockcraft_yaml_base>` is extremely useful when
+the goal is to build a lean **chiselled rock**. Such rocks are typically meant
+for production environments, and as such, the combination of the ``bare``
+:ref:`base <rockcraft_yaml_base>`
+with :ref:`Chisel <chisel_explanation>` slices will result in a minimalistic
+container image that has a reduced attack surface while retaining its
+functionality.
```

Comment:
perhaps words on why these rocks are meant for production environments, or perhaps preferred is the better word.

I would go along the lines of reducing the surface area and the security vectors

Reply:
Sure. Re-worded it